### PR TITLE
Add/code-of-conduct-th.md

### PR DESCRIPTION
**Pull request by minnyww**
## What does this PR do?
Add resource(s) |

## For resources
Code Of Conduct Thai Language

### Description
Add Code Of Conduct Thai Language 

### Why is this valuable (or not)?
Yes, Because help thai people understand what code of conduct in this repo mean.

### How do we know it's really free?
yes is free

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by eshellman** [comment]
super - can you add a link to it as well? in docs/README.md

**Comment by minnyww** [comment]
> super - can you add a link to it as well? in docs/README.md

i already update readme

File: docs/CODE_OF_CONDUCT-th.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+รายงานโดยติดต่อผู้ดูแลโครงการที่ victorfelder ที่ gmail.com ทั้งหมด
+ข้อร้องเรียนจะได้รับการตรวจสอบและสอบสวนและจะส่งผลให้เกิดการตอบสนองที่
+ถือว่าจำเป็นและเหมาะสมกับสถานการณ์ ผู้ดูแลคือ
+มีหน้าที่ต้องรักษาความลับเกี่ยวกับนักข่าวของเหตุการณ์นั้นๆ
+
+
+จรรยาบรรณนี้ดัดแปลงมาจาก [Contributor Covenant][หน้าหลัก],
+version 1.3.0, พร้อมแล้วที่ https://contributor-covenant.org/version/1/3/0/
+
+[หน้าหลัก]: https://contributor-covenant.org
```
Comment (davorpa) [inline docs/CODE_OF_CONDUCT-th.md:45]: There are some parts untranslatable

```suggestion
จรรยาบรรณนี้ดัดแปลงมาจาก [Contributor Covenant][homepage],
version 1.3.0, พร้อมแล้วที่ https://contributor-covenant.org/version/1/3/0/

[homepage]: https://contributor-covenant.org
```

**Review by davorpa — APPROVED** [review]
LGTM

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**